controllers/project.lua

```diff
@@ -146,13 +146,8 @@ ProjectController = {
             -- Description: Get a particular project.
             --              Response will depend on query issuer permissions.
             -- Parameters:  delta, ispublic, ispublished
-            local project =
-                Projects:find(self.params.username, self.params.projectname)
-
-            if not project then yield_error(err.nonexistent_project) end
-            if not (project.ispublic or users_match(self)) then
-                assert_admin(self, err.nonexistent_project)
-            end
+            local project = assert_project(self, self.params.username, self.params.projectname)
```

Comment:
I think this the recommended lapis patter. It gets rid of a couple lines of code whenever we need to find a project.